### ISSUE TITLE
WCAG 1.3.1,WCAG 4.1.2: Ensures select element has an accessible name (#__ac-bf0b9722-46be-425f-a3e3-47cf5c57823e)

### ISSUE THREAD

**Issue by corinagum** [microsoft/BotFramework-WebChat#3815]
### Description: 
Some Adaptive Cards in MockBot need labels added to inputs (available due to AC bump). Note that this is a MockBot change, not Web Chat

### Repro:
- card breakfast
   - choiceset
- card inputs
   - choiceset
- card reminder

#### Issue
`Ensures select element has an accessible name` ([select-name](https://dequeuniversity.com/rules/axe/4.1/select-name?application=msftAI))
#### Target application
[React App](https://corinagum.github.io/WebChat-release-testing/01.create-react-app/index.html)
#### Element path
#__ac-bf0b9722-46be-425f-a3e3-47cf5c57823e
#### Snippet
`<select class="ac-input ac-multichoiceInput ac-choiceSetInput-compact" id="__ac-bf0b9722-46be-425f-a3e3-47cf5c57823e" style="width: 100%; min-width: 0px;">`
#### How to fix
    Fix any of the following:
      aria-label attribute does not exist or is empty
      aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
      Form element does not have an implicit (wrapped) <label>
      Form element does not have an explicit

**Comment by corinagum** [comment]
MockBot has been updated.
